app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,16 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, hummer_level: int, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self._hummer_level = hummer_level
+
+    def player_info(self) -> str:
+        """Get player info."""
+        return (f"Dwarf warrior {self.nickname}. {self.nickname} "
+                f"has a hummer of the {self._hummer_level} level")
```

Comment:
The variable name 'hummer_level' is used here. If it is a typo, it should be corrected to 'hammer_level' to match the intended meaning.